src/platform/github.rs

```diff
@@ -74,3 +74,20 @@ impl Fork for GitHub {
             .to_string())
     }
 }
+
+#[async_trait]
+impl Browse for GitHub {
+    async fn get_browsable_url(&self, url: &Url) -> Result<Url> {
+        let url = Url {
+            scheme: Scheme::Https,
+            raw: Some(format!(
+                "https://{}/{}/{}",
+                url.host.to_string(),
+                url.owner,
+                url.repo
+            )),
+            ..url.clone()
+        };
```

Comment:
I think we don't need to use `Url` type here.
The `Url` type on this crate is for Git repositories, not for general Web.
`url::Url` should be used instead of this, or `format!("https://{}/{}/{}", &self.host, &url.owner, &url.repo)` is enough.

Reply:
gotcha